src/api/java/appeng/api/networking/ticking/IGridTickable.java

```diff
@@ -73,5 +75,6 @@
 	 *
 	 * @return tick rate adjustment.
 	 */
-	TickRateModulation tickingRequest( IGridNode node, int TicksSinceLastCall );
+	@Nonnull
+	TickRateModulation tickingRequest( @Nonnull IGridNode node, int TicksSinceLastCall );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)